src/main/java/org/folio/dew/batch/bursarfeesfines/service/BursarTokenFormatter.java

```diff
@@ -172,6 +172,21 @@ private static String formatUserDataToken(
       BursarExportTokenUserData.ValueEnum.PATRON_GROUP_ID
     ) {
       result = user.getPatronGroup();
+    } else if (
+      tokenUserData.getValue() ==
+      BursarExportTokenUserData.ValueEnum.EXTERNAL_SYSTEM_ID
+    ) {
+      result = user.getExternalSystemId();
+    } else if (
+      tokenUserData.getValue() ==
+      BursarExportTokenUserData.ValueEnum.CORNELL_EXTERNAL_SYSTEM_ID
```

Comment:
What's this for? This looks like it's the same as `EXTERNAL_SYSTEM_ID`; if this is part of the migration then can we just map to `EXTERNAL_SYSTEM_ID` for the same effect, with a `lengthControl` with length 7 and `direction: RIGHT`?

Reply:
Looking at it, it seems like we have inconsistent use of `lengthControl` across these tokens — can you check that there aren't any where its missing?

Reply:
My bad, I thought I already responded to this. We could use length control to extract the part of the system id we need but the format requires that we also pad the truncated id with blanks at the end. I did also notice that not all tokens have length control. I assumed there might have been a reason for that. Here are the data tokens without length control: 
1. FeeMetadata
2. Constant

Reply:
@GoZaddy length control allows truncation and padding, so that should be ok.  Good catch on fee metadata, let's add length control there; constant shouldn't need one since, if they want it to be a certain length, they can just do that

Reply:
Just need to clarify, length-control only truncates when the length of the token is less than specified length in the length-control, right?

Reply:
@GoZaddy correct, if length is too small it'll pad it, if it's too big it'll truncate

Reply:
Yeah, but length-control doesn't support truncate and padding at the same time. According to the format specification  in https://docs.google.com/document/d/1NSQkBFouitZ63HaFPy89BuwKVY1qafctHHiHQ7nuM4c/edit:

"1-11 = the last 7 digits of the patron’s External System ID, which matches their Cornell ID (EmplID), right-padded with blanks. "

So it seems like we're supposed to truncate the external system id (from the left direction) and then pad it with blanks(from the right direction) which length-control doesn't seem to support. That's why I added the `CORNELL_EXTERNAL_SYSTEM_ID` option

Reply:
Ahh, ugh, love absurdly niche things like this.  How about we do:

- truncate external system ID to last 7 digits
- four space constant `    `

Reply:
Yeahh, that definitely works as well. I'll do that!